<ui version="4.0" >
 <class>Sys_Output_Logfile_Dia</class>
 <widget class="QDialog" name="Sys_Output_Logfile_Dia" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>176</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>370</width>
    <height>170</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>500</width>
    <height>176</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Output Logfile</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Output Logfile</string>
       </property>
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="typeLabel" >
          <property name="text" >
           <string>Type:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2" >
         <widget class="QLineEdit" name="typeLineEdit" />
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="pathLabel" >
          <property name="text" >
           <string>Path:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLineEdit" name="pathLineEdit" />
        </item>
        <item row="1" column="2" >
         <widget class="QPushButton" name="browseButton" >
          <property name="text" >
           <string>Browse</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" colspan="2" >
         <widget class="QLineEdit" name="nameLineEdit" />
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2" >
       <item row="0" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>171</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1" >
        <widget class="QPushButton" name="okButton" >
         <property name="text" >
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QPushButton" name="cancelButton" >
         <property name="text" >
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
